openpype/hosts/flame/hooks/pre_flame_setup.py

```diff
@@ -55,43 +56,58 @@ def execute(self):
             "FieldDominance": "PROGRESSIVE"
         }
 
-
         data_to_script = {
             # from settings
-            "host_name": os.getenv("FLAME_WIRETAP_HOSTNAME") or hostname,
-            "volume_name": os.getenv("FLAME_WIRETAP_VOLUME"),
-            "group_name": os.getenv("FLAME_WIRETAP_GROUP"),
+            "host_name": _env.get("FLAME_WIRETAP_HOSTNAME") or hostname,
+            "volume_name": _env.get("FLAME_WIRETAP_VOLUME"),
+            "group_name": _env.get("FLAME_WIRETAP_GROUP"),
             "color_policy": "ACES 1.1",
 
             # from project
             "project_name": project_doc["name"],
             "user_name": user_name,
             "project_data": project_data
         }
-        app_arguments = self._get_launch_arguments(data_to_script)
 
-        self.log.info(pformat(dict(self.launch_context.env)))
+        self.log.info(pformat(dict(_env)))
+        self.log.info(pformat(data_to_script))
 
-        opflame.setup(self.launch_context.env)
+        # add to python path from settings
+        self._add_pythonpath()
+
+        app_arguments = self._get_launch_arguments(data_to_script)
+
+        opfapi.setup(self.launch_context.env)
 
         self.launch_context.launch_args.extend(app_arguments)
 
+    def _add_pythonpath(self):
+        pythonpath = self.launch_context.env.get("PYTHONPATH")
+
+        # separate it explicity by `;` that is what we use in settings
+        new_pythonpath = self.flame_pythonpath.split(";")
```

Comment:
Shouldn't this be `os.pathsep`?

Reply:
I believe it could but I believe linux or macos is having ":" and could be different from what we have in settings, right?

Reply:
Environments are joined with `os.pathsep` (when have list of values) so this would work only on windows.

Reply:
added in gitkraken://repolink/33c731ef76967f4bd3f6ba8b287ecc0c52412f33/commit/95be2c3bc8fc5a91ac2f65072fd56a6a30cda872?url=https%3A%2F%2Fgithub.com%2Fpypeclub%2FOpenPype.git